MonoGame.Framework/Platform/SDL/SDL2.cs

```diff
@@ -1106,4 +1111,20 @@ public static void UpdateEffect(IntPtr haptic, int effect, ref Effect data)
             GetError(SDL_HapticUpdateEffect(haptic, effect, ref data));
         }
     }
+
+    public static class Drop
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public unsafe struct Event
+        {
+            public EventType Type;
+            public uint TimeStamp;
+            public char* File;
```

Comment:
You can use IntPtr here to avoid unsafe, the remaining code already uses it so we should probably stick with it for now.

Reply:
`char*` is actually confusing as the underlying data is UTF8. `IntPtr` and `byte*` are the correct types.